assets/css/_sass/website/_shame.scss

```diff
@@ -15,4 +15,20 @@
   font-size: 0.75rem;
   font-weight: 400;
   text-transform: uppercase;
+}
+
+/* 
+* This will be removed when we move to version to a new version of the helix-ui code that supports the card component
+*/
+
+
+.resource-card {
+	border: 1px solid #e0e0e0;
+	padding: 20px 20px;
+	background: #fafafa;
+    &:hover {
+      -webkit-box-shadow: 0px 3px 3px 0px rgba(0,0,0,0.16);
```

Comment:
Use the box-shadow mixin: https://github.com/rackerlabs/design-system/blob/staging/assets/css/_sass/website/_mixins.scss#L1

Reply:
updated. Didn't know we had this. thanks!